docs/source/services/timelock_service/client_configuration.rst

```diff
@@ -0,0 +1,54 @@
+.. _timelock-client-configuration:
+
+Timelock Client Configuration
+=============================
+
+.. danger::
+
+   Improperly configuring one's cluster to use external timestamp and lock services can result in **SEVERE DATA
+   CORRUPTION**! Please contact the AtlasDB team before you configure your clients to use this.
+
+You will need to update your AtlasDB configuration in order to have said clients request timestamps and locks from
+external Timelock Servers as opposed to their embedded services. This is an extension of the leader block configuration
+options discussed at :ref:`leader-config`.
+
+Instead of configuring a ``leader`` block, or both a ``timestamp`` and ``lock`` block, one may instead specify a
+single ``timelock`` block.
+
+The Timelock Block
+------------------
+
+The ``cluster`` block is used to identify the servers which make up a Timelock Service cluster. An example is as
+follows:
+
+   .. code:: yaml
+
+      timelock:
```

Comment:
We should add another example to the [leader configuration page](https://palantir.github.io/atlasdb/html/configuration/leader_config.html#leader-configuration-examples) that includes this as an option (with an "experimental" in parentheses).